lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -11,10 +17,31 @@
     </div>
 </div>
 
-<div class="form-instructions create-team-instructions">
-    <p class="copy">
-        <%- gettext("Enter information to describe your team. You cannot change these details after you create the team.") %></p>
-</div>
+<% if (action === 'create') { %>
+    <div class="form-instructions create-team-instructions">
+        <p class="copy">
+            <%- gettext("Enter information to describe your team. You cannot change these details after you create the team.") %></p>
+    </div>
+<% } else if (action === 'edit') { %>
+    <div class="team-info wrapper-msg-fixed is-incontext urgency-low warning" tabindex="-1">
+        <div class="msg">
+            <div class="msg-content">
+                <div class="copy">
+                <p>
+                    <%=
+                        interpolate_text(
+                            _.escape(gettext("The team that you are editing has {total_members} members. Notify team members prior to making significant changes.")),
```

Comment:
@muzaffaryousaf I just noticed that this always says "members" even when the team member count is 1 ("...has 1 members."). Is there a way to make the noun agree with the number?